src/pal/inc/pal.h

```diff
@@ -4600,13 +4601,14 @@ PALIMPORT char * __cdecl _strdup(const char *);
 PALIMPORT PAL_NORETURN void __cdecl exit(int);
 int __cdecl atexit(void (__cdecl *function)(void));
 
-PALIMPORT DLLEXPORT void __cdecl qsort(void *, size_t, size_t, int (__cdecl *)(const void *, const void *));
-PALIMPORT void * __cdecl bsearch(const void *, const void *, size_t, size_t,
-int (__cdecl *)(const void *, const void *));
-
 PALIMPORT char * __cdecl _fullpath(char *, const char *, size_t);
 
 #ifndef PAL_STDCPP_COMPAT
+
+PALIMPORT DLLEXPORT void __cdecl qsort(void *, size_t, size_t, int(__cdecl *)(const void *, const void *));
+PALIMPORT DLLEXPORT void * __cdecl bsearch(const void *, const void *, size_t, size_t,
```

Comment:
Could these be `PALIMPORT PALAPI void bsearch/qsort/getenv/_putenv(..`, as `PALAPI` is defined as `DLLEXPORT __cdecl`?

Reply:
The pattern used to export C-runtime APIs is `PALIMPORT DLLEXPORT`, so I kept this one for consistency. 

C-runtime APIs were not PALAPIs originally when we had a thin (just forwarders) PAL layer for Windows as well. On Windows, the C-runtime APIs were cdecl, but PALAPIs were stdcall.